ethstats/ethstats.go

```diff
@@ -1035,8 +1061,8 @@ func (s *Service) reportStats(conn *websocket.Conn) error {
 	log.Trace("Sending node details to ethstats")
 
 	stats := map[string]interface{}{
-		"id":      s.node,
-		"address": etherBase,
+		"id":      s.backend.ValidatorAddress().String(),
+		"address": validatorAddress,
```

Comment:
if eth == nil validatorAddress is empty. Can s.eth be nil? shouldn't that imply we need to skip or something?